itest/lnd_sweep_test.go

```diff
@@ -2352,3 +2352,177 @@ func testFeeReplacement(ht *lntest.HarnessTest) {
 	// Finally, clean the mempool.
 	ht.MineBlocksAndAssertNumTxes(1, 1)
 }
+
+// testBumpFeeLowBudget checks that when the requested ideal budget cannot be
+// met, the sweeper still sweeps the input with the actual budget.
+func testBumpFeeLowBudget(ht *lntest.HarnessTest) {
+	// Create a new node with a large `maxfeerate` so it's easier to run the
+	// test.
+	alice := ht.NewNode("Alice", []string{
+		"--sweeper.maxfeerate=10000",
+	})
+
+	// Fund Alice 2 UTXOs, each has 100k sats. One of the UTXOs will be used
+	// to create a tx which she sends some coins to herself. The other will
+	// be used as the budget when CPFPing the above tx.
+	walletBalance := btcutil.Amount(100_000)
```

Comment:
Q: why is this called walletBalance if the wallet is funded with 2 times this amount ?

Reply:
The rest are used as mining fees,
> The other will be used as the budget when CPFPing the above tx.